.github/workflows/ci.yml

```diff
@@ -0,0 +1,305 @@
+name: Rust
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - develop-ci # TODO: remove
```

Comment:
Can you remove this, please?

Reply:
Ok I remember, I remove it just before merge.

Reply:
Done